src/views/common/DashOverview.tsx

```diff
@@ -19,6 +27,28 @@ export const DashOverviewIntro: React.FC<{
   balance: BigNumber | undefined;
 }> = ({ asset, mode, onSubmit, amount, setAmount, balance }) => {
   const { data: decimals } = useDecimalCountForToken(asset.tokenAddress);
+  const { data: wNative } = useWrappedNativeDefinition();
+  const tokenAddress =
+    asset.tokenAddress === NATIVE_TOKEN
+      ? wNative?.tokenAddress
+      : asset.tokenAddress;
+
+  const newHealthFactor = useNewHealthFactorCalculator(
+    amount,
+    tokenAddress,
+    mode === "withdraw" || mode === "deposit" ? true : false,
+    mode === "deposit" || mode === "borrow" ? true : false
```

Comment:
`x ? true : false` always simplifies to `x`
 
```suggestion
    mode === "withdraw" || mode === "deposit",
    mode === "deposit" || mode === "borrow"
```